Library/KeyValueStoreType.swift

```diff
@@ -4,6 +4,9 @@ public enum AppKeys: String {
   // swiftformat:disable wrap
   case closedFacebookConnectInActivity = "com.kickstarter.KeyValueStoreType.closedFacebookConnectInActivity"
   case closedFindFriendsInActivity = "com.kickstarter.KeyValueStoreType.closedFindFriendsInActivity"
+  case commentFlaggingEnabled = "com.kickstarter.KeyValueStoreType.iOSCommentThreadingCommentFlagging"
```

Comment:
I wonder if instead of persisting these separately we could just persist a dictionary 🤔 It may make future changes to flags simpler? Alternatively a `struct` or something conforming to `Decodable` could be persisted as a `Data` type in user defaults.

Reply:
Opted for the dictionary here and updated!